README.md

```diff
@@ -1,5 +1,7 @@
 [![Code Climate](https://codeclimate.com/github/alexandru-calinoiu/fog-akamai/badges/gpa.svg)](https://codeclimate.com/github/alexandru-calinoiu/fog-akamai)
 
+[![Test Coverage](https://codeclimate.com/github/alexandru-calinoiu/fog-akamai/badges/coverage.svg)](https://codeclimate.com/github/alexandru-calinoiu/fog-akamai/coverage)
```

Comment:
Same thing here. You are pointing your  user when it should be the fog organization. Could you fix it please?

Thanks!